fabric-screen-handler-api-v1/src/main/java/net/fabricmc/fabric/api/screenhandler/v1/SlotWithClickAction.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.screenhandler.v1;
+
+import net.minecraft.entity.player.PlayerInventory;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ClickType;
+
+/**
+ * An interface for slots that consume a click action.
```

Comment:
`Slot` should be linked here (especially since 1.17 adds that entity command item slot as well)

Reply:
Good call

Reply:
well, the fact that screen handler's slot and commands' item slots are distinct systems internally (though often times closely related) already can be confusing. So linking is a good idea.